classes/class-loader.php

```diff
@@ -95,7 +95,10 @@ function ( $class_name ) {
 					if ( false !== strpos( $class_name, 'P4GBKS' ) ) {
 						$class_name_parts = explode( '\\', $class_name );
 						$real_class_name  = array_pop( $class_name_parts );
-						$file_name        = 'class-' . str_ireplace( '_', '-', strtolower( $real_class_name ) );
+						if ( 'covers' === $real_class_name ) {
```

Comment:
Probably not needed, since currently `$real_class_name` doesn't match this and it still works :smile: